release-1.6/release-notes-draft.md

```diff
@@ -58,12 +58,33 @@
 
 ## Cluster Lifecycle
 ### New Features
+- kubeadm is a simple tool for creating Kubernetes clusters on existing
+  machines. kubeadm's command line interface is now in beta. This means that
+  using it is considered safe.
```

Comment:
beta means that you won't change the API without good reason, I think - nothing to do with whether it works or not, for example.  Perhaps drop "This means that" and just go with "Using it is considered safe".

I would still say that a cautionary note is in order here - like "no upgrades, no HA, no cloudprovider support", but OTOH you do have the feedback link in there... On that note, can you please update the link to point to #kubeadm so we aren't overwhelmed in #sig-cluster-lifecycle :-)